subsys/bluetooth/services/fast_pair/fp_battery.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <bluetooth/services/fast_pair.h>
+#include "fp_battery.h"
+
+static struct bt_fast_pair_battery_data fp_battery_data = {
+	.battery_values = {{ .charging = false, .level = BT_FAST_PAIR_UNKNOWN_BATTERY_LEVEL},
+			   { .charging = false, .level = BT_FAST_PAIR_UNKNOWN_BATTERY_LEVEL},
+			   { .charging = false, .level = BT_FAST_PAIR_UNKNOWN_BATTERY_LEVEL}}
+};
+
+static bool validate_battery_value(struct bt_fast_pair_battery_value battery_value)
```

Comment:
Can we name it `battery_value_valid`? That would indicate that we return bool (in many places we return integer for similar use-cases and then logic is inverted).